src/main/java/com/myhome/controllers/CommunityController.java

```diff
@@ -206,4 +206,3 @@ public CommunityController(
     }
   }
 }
```

Comment:
@Sountastic A corresponding `@DeleteMapping` has to be present in the controller. It seems to be missing.